hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
```

Comment:
Should be pretty easy to figure out what the impact would be: (removing obvious false positives)
```console
$ git grep -n VERSION -- '*.go'
api/client/hijack.go:152:       req.Header.Set("User-Agent", "Docker-Client/"+dockerversion.VERSION)
api/client/utils.go:67: req.Header.Set("User-Agent", "Docker-Client/"+dockerversion.VERSION)
api/client/version.go:25:       if dockerversion.VERSION != "" {
api/client/version.go:26:               fmt.Fprintf(cli.out, "Client version: %s\n", dockerversion.VERSION)
api/common.go:19:       APIVERSION            version.Version = "1.19"       // Current REST API version
api/server/server.go:248:               Version:    dockerversion.VERSION,
daemon/daemon.go:844:   localCopy := path.Join(config.Root, "init", fmt.Sprintf("dockerinit-%s", dockerversion.VERSION))
docker/daemon.go:113:           Version:     dockerversion.VERSION,
docker/daemon.go:156:           "version":     dockerversion.VERSION,
docker/docker.go:164:   fmt.Printf("Docker version %s, build %s\n", dockerversion.VERSION, dockerversion.GITCOMMIT)
graph/graph.go:127:             DockerVersion: dockerversion.VERSION,
graph/graph_test.go:110:        if img.DockerVersion != dockerversion.VERSION {
graph/graph_test.go:111:                t.Fatalf("Wrong docker_version: should be '%s', not '%s'", dockerversion.VERSION, img.DockerVersion)
integration-cli/docker_api_version_test.go:22:  if v.Version != dockerversion.VERSION {
integration-cli/docker_cli_save_load_test.go:292:               exec.Command("grep", "VERSION"),
registry/registry.go:47:        httpVersion = append(httpVersion, useragent.VersionInfo{"docker", dockerversion.VERSION})
```